doc/source/ecosystem.rst

```diff
@@ -38,7 +38,10 @@ Statsmodels leverages pandas objects as the underlying data container for comput
 Use pandas DataFrames in your `scikit-learn <http://scikit-learn.org/>`__
 ML pipeline.
 
+`Featuretools <https://github.com/featuretools/featuretools/>`__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Featuretools is a Python library for automated feature engineering built on top of pandas. It excels at transforming transactional and relational datasets into feature matrices for machine learning using reusable feature engineering "primitives". Users can contribute their own primitives in python and share them with the rest of the community. 
```

Comment:
What is meant with "transactional" in this context?

Reply:
it is being used as a synonym for "time series". we use the word transactional here to highlight that the data could be an irregular time series. for example, a series of customer behaviors on a website. happy to make it just time series in this description.